### DISCUSSION TITLE
Has anyone been able to cross-compile from x64-windows to x64-android?

### DISCUSSION THREAD

**Discussion by pedroarroyo** [microsoft/vcpkg#32148] [Q&A]
Has anyone been able to cross-compile from x64-windows to x64-android?  Cross compilation seems to be failing for me due to running malloc tests.

-- Performing Test HAVE_MM_MALLOC
Access violation
ninja: build stopped: subcommand failed.

Alternatively, would anyone know if it's easy to disable these malloc tests so that the build can continue?

**Comment by dg0yt** [comment]
> Has anyone been able to cross-compile from x64-windows to x64-android? 

Yes: you. (And others, too). Otherwise you wouldn't even get close to building pcl.

> Cross compilation seems to be failing for me due to running malloc tests.

Leave that discussion in #32136.
